neon_core/configuration/neon.yaml

```diff
@@ -115,15 +114,16 @@ hotwords:
     lang: en
     samples:
       - hey neon
-  thank you:
+  thank_you:
     module: ovos-ww-plugin-vosk
     sound: snd/acknowledge.mp3
     active: false
-  wake up:
+  wake_up:
     module: ovos-ww-plugin-vosk
     active: false
```

Comment:
this will explicitly disable wake_up and may lock a device in sleep mode, it should not be set here, this will stop the listener section of the config from loading the default standup_word

Reply:
Makes sense; I don't think Neon has a method for entering sleep mode currently, but I should keep the config around for compat.

Reply:
it can be done via bus messages, or if a user installs naptime skill via voice. the functionality itself is part of the listener code, so the mechanism to bail out should also be there by default imho